cmd/block-generator/generator/generate.go

```diff
@@ -450,33 +451,44 @@ func (g *generator) generatePaymentTxn(round uint64, intra uint64) (transactions
 	return g.generatePaymentTxnInternal(selection.(TxTypeID), round, intra)
 }
 
+const minbal = uint64(100000)
```

Comment:
nit: please caps for const.

Reply:
I don't want to export it, but I probably should have at least used camelcase